addon/metrics-adapters/google-tag-manager.js

```diff
@@ -20,7 +20,7 @@ export default BaseAdapter.extend({
   init() {
     const config = get(this, 'config');
     const { id, envParams } = config;
-    const dataLayer = getWithDefault(config, 'dataLayer', 'dataLayer');
+    const dataLayer = get(config, 'dataLayer') || 'dataLayer';
```

Comment:
The `dataLayer` can not validly be any falsey values, so a simple `||` is sufficient.